Open Judge System/Services/OJS.Services/Business/ParticipantScores/Models/CategoryContestsParticipationSummaryExcel.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.IO;
+using MissingFeatures;
+using NPOI.HSSF.UserModel;
+using NPOI.SS.UserModel;
+using OJS.Web.Common.Extensions;
+
+namespace OJS.Services.Business.ParticipantScores.Models
+{
+    public class CategoryContestsParticipationSummaryExcel
+    {
+        public HSSFWorkbook Workbook { get; set; }
+        
+        public CategoryContestsParticipationSummary CategorySummary { get; set; }
+        
+        private int rowNumber = 1;
+        private ISheet sheet;
+
+        public CategoryContestsParticipationSummaryExcel(CategoryContestsParticipationSummary categorySummary)
+        {
+            this.CategorySummary = categorySummary;
+            Create();
+        }
+        
+        public void Create()
+        {
+            this.Workbook = new HSSFWorkbook();
+            this.sheet = Workbook.CreateSheet();
+            var columnsCount = CreateResultsSheetHeaderRow();
+
+            FillSheet();
+
+            sheet.AutoSizeColumns(columnsCount);
+        }
+
+        public MemoryStream GetAsStream()

```

Comment:
All methods are `public`. Is this correct?

Reply:
No methods beside Create (which does not have to be and is already fixed) and GetAsStream which is needed to be such are private, they are just missing the private keyword.